meteor/server/api/ingest/generation.ts

```diff
@@ -274,11 +268,6 @@ export async function calculateSegmentsFromIngestData(
 					)
 				)
 			})
-
-			// If the segment has no parts, then hide it
-			if (blueprintSegment.parts.length === 0) {
-				newSegment.isHidden = true
```

Comment:
I understand that this removal refers to "The on-air segment can be hidden if blueprints return an empty array of parts (e.g. the segment is "floated" in an NRCS while on-air).", but it seems like this should be handled some other way then? I really don't think we want to have empty Segments visible in the GUI.

Reply:
This was also moved to the commit stage https://github.com/nrkno/tv-automation-server-core/pull/638/files#diff-432a5ee01b84991e0799a0ccb9960dbad910336d7a5f4a5494feb0fb1f971e07R281